src/app.module.ts

```
@@ -37,7 +40,17 @@ dotenv.config();
         username: configService.get<string>('DB_USERNAME'),
         password: configService.get<string>('DB_PASSWORD'),
         database: configService.get<string>('DB_NAME'),
-        entities: [User, SavedPost, Post, Application, Message, Comment, Job, Portfolio],
+        entities: [
+          User, 
```

ESLint

[error] Delete `·`

(prettier/prettier)